common/oidc-client/oidc-authservice/base/statefulset.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: authservice
       containers:
       - name: authservice
-        image: gcr.io/arrikto/kubeflow/oidc-authservice:e236439
```

Comment:
we'll need to not touch this at all, in order for the image transformer to use `name` and detect the old arrikto image and then `newName` to convert to the new name

Reply:
Changed, thanks!